src/engines/kotor2/module.cpp

```diff
@@ -318,6 +319,12 @@ void Module::enter() {
 		_pc->show();
 	}
 
+	Graphics::Aurora::Model *model = _pc->getModel().get();
+	model->clearDefaultAnimations();
+	model->addDefaultAnimation(Common::UString("pause3"), 25);
+	model->addDefaultAnimation(Common::UString("pause2"), 25);
+	model->addDefaultAnimation(Common::UString("pause1"), 50);
```

Comment:
Btw, I'm not really a fan of having this getModel() method, here and in KotOR1. The Creature itself should add these default animations when a PC is created.

Likewise, there should be a playAnimation() method (see NWN does it, for example), and maybe even run()/stop() methods for running/stopping.